### PR TITLE
Add tags to each story and show that in the sidebar

### LINKED ISSUE #1247: the status of the components (important to show status of components that are experimental/drafts)

### PR DESCRIPTION
**Pull request by jpzwarte**
Fixes #1247

### CODE REVIEW

File: packages/angular/package.json [hunk 1]
```diff
@@ -90,7 +90,7 @@
     "karma-jasmine-html-reporter": "~2.1.0",
     "ng-packagr": "^17.3.0",
     "rxjs": "~7.8.1",
-    "storybook": "^8.0.10",
+    "storybook": "^8.1.1",
```
Comment (anna-lach) [inline packages/angular/package.json:93]: Are we going to add tags to Angular wrappers? Or it's rather a separated category, so I'm not sure how we should treat those elements? :)

image

File: packages/components/grid/src/stories/basics.stories.ts [hunk 2]
```diff
@@ -10,7 +10,7 @@ import { type GridColumnDataRenderer } from '../column.js';
 type Story = StoryObj;
 
 export default {
-  title: 'In progress/Grid/Basics'
+  title: 'Layout/Grid/Basics'
```
Comment (anna-lach) [inline packages/components/grid/src/stories/basics.stories.ts:13]: Do we want to set `draft` status for grid as well? :)

**Review by anna-lach — APPROVED** [review]

File: packages/angular/package.json (hunk 1 above)
Comment (jpzwarte) [reply packages/angular/package.json:93]: I think the angular wrappers should reflect the web components they're based on.

File: packages/components/grid/src/stories/basics.stories.ts (hunk 2 above)
Comment (jpzwarte) [reply packages/components/grid/src/stories/basics.stories.ts:13]: Grid is a separate folder atm. I'd have to look into that.

**Pull request merged**